archive/src/main/java/io/orkes/conductor/dao/postgres/archive/PostgresArchiveDAO.java

```diff
@@ -41,48 +41,48 @@
 
     private static final String REMOVE_WORKFLOW =
-            "DELETE FROM workflow_archive WHERE workflow_id = ?";
+            "DELETE FROM archive.workflow_archive WHERE workflow_id = ?";
     public static final String UNKNOWN = PostgresArchiveDAO.class.getName() + "_UNKNOWN";
     public static final String GET_WORKFLOW_FAMILY = "WITH RECURSIVE workflow_hierarchy AS (" +
             "   SELECT" +
-            "       workflow_id, parent_workflow_id, json_data, status from workflow_archive where workflow_id = ?" +
+            "       workflow_id, parent_workflow_id, json_data, status from archive.workflow_archive where workflow_id = ?" +
             "   UNION ALL" +
             "   SELECT child_wf.workflow_id workflow_id, child_wf.parent_workflow_id, child_wf.json_data, child_wf.status" +
-            "   FROM workflow_archive AS child_wf" +
+            "   FROM archive.workflow_archive AS child_wf" +
             "      JOIN workflow_hierarchy parent_wf ON child_wf.parent_workflow_id = parent_wf.workflow_id" +
             ")\n" +
             "SELECT workflow_id, parent_workflow_id, json_data, status FROM workflow_hierarchy;";
     public static final String GET_WORKFLOW_FAMILY_SUMMARY = "WITH RECURSIVE workflow_hierarchy AS (" +
             "   SELECT" +
-            "       workflow_id, parent_workflow_id, status from workflow_archive where workflow_id = ?" +
+            "       workflow_id, parent_workflow_id, status from archive.workflow_archive where workflow_id = ?" +
             "   UNION ALL" +
             "   SELECT child_wf.workflow_id workflow_id, child_wf.parent_workflow_id, child_wf.status" +
-            "   FROM workflow_archive AS child_wf" +
+            "   FROM archive.workflow_archive AS child_wf" +
             "      JOIN workflow_hierarchy parent_wf ON child_wf.parent_workflow_id = parent_wf.workflow_id" +
             ")\n" +
             "SELECT workflow_id, parent_workflow_id, status FROM workflow_hierarchy;";
     public static final String GET_WORKFLOW_PATH = "WITH RECURSIVE workflow_path AS (" +
             "   SELECT" +
-            "       workflow_id, parent_workflow_id from workflow_archive child_wf where workflow_id = ?" +
+            "       workflow_id, parent_workflow_id from archive.workflow_archive child_wf where workflow_id = ?" +
             "   UNION ALL" +
             "   SELECT parent_wf.workflow_id workflow_id, parent_wf.parent_workflow_id" +
-            "   FROM workflow_archive AS parent_wf" +
+            "   FROM archive.workflow_archive AS parent_wf" +
             "      JOIN workflow_path child_wf ON child_wf.parent_workflow_id = parent_wf.workflow_id" +
             ")\n" +
             "SELECT workflow_id, parent_workflow_id FROM workflow_path;";
 
     private final DataSource searchDatasource;
 
-    private static final String TABLE_NAME = "workflow_archive";
+    private static final String TABLE_NAME = "archive.workflow_archive";
```

Comment:
This constant is a little use less since it is not used in most of the SQL queries, feel free to remove it

Reply:
removed